examples/balance/app.ddlog

```diff
@@ -0,0 +1,12 @@
+yes (vote_id bigint).
+no  (vote_id bigint).
+
+whatever? (
+    candidate bigint
+).
+
+@weight(1)
+whatever(p) :- yes(_).
```

Comment:
OK, I don't think it's necessary to make `yes` / `no` variables or use IMPLY. The ratio semantics concern the counts of "positive" and "negative" factors.